app/Http/Controllers/API/V1/TagController.php

```diff
@@ -7,17 +7,21 @@
 use Illuminate\Http\Request;
 
 class TagController extends Controller
-{ public function index(){
+{ public function index()
+{
     return Tag::all();
 }
-    public function show( Tag $tag){
+    public function show( Tag $tag)
+    {
         return $tag;
     }
-    public function store( Request $request){
+    public function store( Request $request)
```

Comment:
Request Form Validations?

Reply:
still didn't do any resources or requests for task and tag models